pyoozie/client.py

```diff
@@ -515,6 +519,36 @@ def job_coordinator_kill(self, coordinator_id=None, name=None, user=None):
             return True
         return False
 
+    def job_coordinator_rerun(self, coordinator_id):
+        coord = self._fetch_coordinator_or_action(coordinator_id)
+        if not coord.is_action():
+            raise ValueError('Rerun only supports coodinator action IDs')
+
+        if not coord.status.is_active():
```

Comment:
We should also check that the coordinator itself IS active (i.e. `coord.coordinator.status.is_active()`); rerun on a killed coord does not work (and leads to the mysterious `DONEWITHERROR`).

See https://oozie.apache.org/docs/4.1.0/DG_CoordinatorRerun.html